src/features/MovieList/MovieTile/index.js

```diff
@@ -14,12 +14,16 @@ import {
     Votes,
     StyledStar,
     DescriptionContainer,
+    ImgWrapper,
 } from "./styled";
 
 export const MovieTile = ({ title, year, tags, rate, votes }) => {
     return (
         <Content>
-            <Img src={Poster} />
+            <ImgWrapper>
+                <Img src={Poster} />
+            </ImgWrapper>
+
             <Wrapper>
                 <DescriptionContainer>
```

Comment:
DescriptionContainer component is not needed. You could remove it from styled.js and imports and replace it with `<div>` on this line

Reply:
problem risole, the descriptionContainer is deleted now